web/src/app/profile/approved-researcher/components/AgreementText.tsx

```diff
@@ -9,7 +9,7 @@ type AgreementTextProps = {
 
 export default function AgreementText(props: AgreementTextProps) {
   return (
-    <section className="section" id="approved-researcher-agreement-text">
+    <section className="section section--approved-researcher-agreement-text">
       <Markdown>{props.text}</Markdown>
     </section>
   );
```

Comment:
Since the HTML already tells us it is a section, this could be simplified to make the BEM relationships work. Also, the `--` should represent visual modifications. You could try something like:

```
<section className="approved-researcher-agreement">
  <h2 className="approved-researcher-agreement__title">...</h2>
  <p className="approved-researcher-agreement__text">...</p>
</section>
```